### PR TITLE
@Injectable() added to top of the Class for IVY Support

### PR DESCRIPTION
**Pull request by mobilerast**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: docs/recipes/cache.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ export class GetNovels {
   name: 'novels',
   defaults: []
 })
+@Injectable
```
Comment (splincode) [inline docs/recipes/cache.md]: `@Injectable()` :)

**Comment by markwhitfeld** [comment]
@mobilerast Thank you so much for the contributions, but please try to roll them into one PR in the future. This just makes it very noisy and takes up more time unnecessarily.
👍

**Comment by debender495** [comment]
what does this means? We don't have to add @Injectable() on every state we create, for Angular 9

**Comment by splincode** [comment]
it's a must for ivy

**Comment by splincode** [comment]
https://www.ngxs.io/advanced/ivy-migration-guide

**Pull request merged**